src/wwwroot/template-editor/adapterService.js

```diff
@@ -0,0 +1,356 @@
+(function() {
+    'use strict';
+  
+    angular
+      .module('mseditor')
+      .service('adapterService', adapterService);
+  
+      adapterService.$inject = [
+      '$http',
+      '$q'
+    ];
+  
+    function adapterService($http, $q) {
+      var service = {
+        getCampaign: getCampaign,
+        campaignSaveChanges: campaignSaveChanges,
+        campaignSaveChangesAsTemplate: campaignSaveChangesAsTemplate,
+        getCampaignRssPreview: getCampaignRssPreview,
+        deleteImages: deleteImages,
+        getImagesForCampaign: getImagesForCampaign,
+        getPreviewImage: getPreviewImage,
+        getPreviewImageWithPlay: getPreviewImageWithPlay,
+        getTiendaNubeProducts: getTiendaNubeProducts,
+        getSettings: getSettings
+      };
+  
+      return service;
+
+      /**
+       * Return a preview placeholder image
+       * @method
+       * @argument {object} params
+       * @example
+       * {
+       *  fileName: fileName,
+       *  providerName: providerName,
+       *  idCampaign: campaign.getIdCampaign(),
+       *  idTemplate: campaign.getIdTemplate()
+       * }
+       */
+      function getPreviewImageWithPlay(params) {
+        console.log("getPreviewImageWithPlay");
+        var response = {
+          ThumbnailUrlWithPlay: 'https://i.vimeocdn.com/filter/overlay?src0=' + 'http://www.wpclipart.com/travel/US_Road_Signs/info/temporary.png' + '&src1=http%3A%2F%2Ff.vimeocdn.com%2Fp%2Fimages%2Fcrawler_play.png'
+        };
+        var fakePromise = $q.resolve(response);
+        return fakePromise;
+      }
+
+      /**
+       * Return a preview image from a video
+       * @method
+       * @argument {object} params
+       * @example
+       * {
+       *   url: url,
+       *   addPlayButton: addPlayButton,
+       *   idCampaign: campaign.getIdCampaign(),
+       *   idTemplate: campaign.getIdTemplate()
+       * }
+       */
+      function getPreviewImage(params) {
+        console.log("getPreviewImage");
+        var response = {
+              ThumbnailUrl: 'https://i.vimeocdn.com/video/699535210.webp',
+              Title: 'Test video',
+              ProviderName: 'Youtube',
+              /** @property {string} [ThumbnailUrlWithPlay] It's added when 'Add reproductor' is checked */
+              ThumbnailUrlWithPlay: 'https://i.vimeocdn.com/filter/overlay?src=' + 'https://i.vimeocdn.com/video/699535210.webp' + '&src=https://integrationstore-b0c3f53658fe7a75.microservice.createsend.com/files/9392B9D9-F380-42FC-9571-7E109B7A1C26/youtube-play-button-overlay.png'
+            };
+        var fakePromise = $q.resolve(response);
+        return fakePromise;
+      }
+
+      /**
+       * Return a preview images to image gallery
+       * @method
+       * @argument {integer} start - Initial position index 
+       * @argument {integer} howMany - Offset position
+       * @argument {object} query - Query model
+       * @example
```

Comment:
Is this _@example_ tag subordinate to the previous _@argument_? does it work in that way?